portfolio/src/main/webapp/script.js

```diff
@@ -78,3 +77,15 @@ function deleteComment(comment) {
   params.append('id', comment.id);
   fetch('/delete-comment', {method: 'POST', body: params});
 }
+
+function loadMap() {
+  console.log('hello');
+  const map = new google.maps.Map(
+    document.getElementById("map"),
+    {center: {lat: 37.422, lng: -122.084}, zoom: 16});
+}
+
+function loadFunctions() {
+  getComments();
```

Comment:
Instead of having a loadFunctions() method, I think you can directly have multiple loading functions in the HTML

`<body onload="func1(); func2();">`
https://www.htmlgoodies.com/beyond/javascript/article.php/3724571/using-multiple-javascript-onload-functions.htm